README.md

```diff
@@ -48,7 +48,9 @@ Follow the step-by-step instructions in this section to build and test the custo
 
 ## 1. Test your REST Endpoint URL
 
-Before running the CloudFormation template, you need an HTTP/HTTPS endpoint to expose your REST resources. Make sure your application conforms to the REST API specification in the [custom-resource-stack.yaml](https://github.com/aws/aws-auto-scaling-custom-resource/blob/master/cloudformation/templates/custom-resource-stack.yaml) CloudFormation template. 
+Before running the CloudFormation template, you need an HTTP/HTTPS endpoint to expose your REST resources. Make sure your application conforms to the REST API specification in the [custom-resource-stack.yaml](https://github.com/aws/aws-auto-scaling-custom-resource/blob/master/cloudformation/templates/custom-resource-stack.yaml) CloudFormation template.  
+
+A sample REST endpoint is provided as a Dockerized Apache Python cgi.   See: [sample-api-server](./sample-api-server/)  
 
 After you create an endpoint that contains the required REST resources, you can verify that the endpoint URL works by issuing GET and PATCH requests to it, for example: 
```

Comment:
Could we say this instead?

Note: If you need an environment to test with, a sample REST endpoint is provided as a Dockerized Apache Python CGI. For more information, see [sample-api-server](./sample-api-server/).

Reply:
sure.  sounds good